tf_yarn/_dispatch_task.py

```diff
@@ -115,13 +117,17 @@ def stop_event(e: typing.Optional[Exception]) -> None:
 
         config = experiment.config
         assert config.task_type == task_type and config.task_id == task_id
-
+    
+    server_config = tf.ConfigProto()
+    server_config.intra_op_parallelism_threads=num_threads
+    server_config.inter_op_parallelism_threads=num_threads
     if fake_google_env:
+
         tf.train.Server(
             config.cluster_spec,
             job_name=config.task_type,
             task_index=config.task_id,
-            config=config.session_config,
+            config=server_config,
```

Comment:
I suggest to keep config.session_config. Otherwise the user can't inject anything anymore from his estimator.

What we could do is to add a flag overrideThreadsParallelismConfig, false by default and we patch the values with out default values. When it is true we use the values specified.
What do you think ?

Reply:
Yes, I am working on this to make sure it doesn't cause any issue.